### ISSUE TITLE
pymc-nightly builds broken

### ISSUE THREAD

**Issue by fonnesbeck** [pymc-devs/pymc#5679]
The nightly builds are currently not running to completion. Can you look into this @theorashid? Looks like it may be authentication-related.

**Comment by theorashid** [comment]
is this the same error I get in #5598 ? Because something is uploading to [pypi](https://pypi.org/project/pymc-nightly/4.0.0b6.dev20220401/)

**Comment by twiecki** [comment]
```
WARNING: Generating metadata for package pymc-nightly produced metadata for project name pymc. Fix your #egg=pymc-nightly fragments.
[17](https://github.com/pymc-devs/pymc/runs/5802654081?check_suite_focus=true#step:5:17)
Discarding https://files.pythonhosted.org/packages/5b/87/f27a1c6408e3c5cb7fd9efbe2c61d333881689a6c381fe780fac788ed2ff/pymc-nightly-4.0.0b6.dev20220403.tar.gz#sha256=c2b51cc9425b583341d0770d034ed300850eca38b52864147091864bfbead0e8 (from https://pypi.org/simple/pymc-nightly/) (requires-python:>=3.7): Requested pymc from https://files.pythonhosted.org/packages/5b/87/f27a1c6408e3c5cb7fd9efbe2c61d333881689a6c381fe780fac788ed2ff/pymc-nightly-4.0.0b6.dev20220403.tar.gz#sha256=c2b51cc9425b583341d0770d034ed300850eca38b52864147091864bfbead0e8 has inconsistent name: filename has 'pymc-nightly', but metadata has 'pymc'
[18](https://github.com/pymc-devs/pymc/runs/5802654081?check_suite_focus=true#step:5:18)
ERROR: Could not find a version that satisfies the requirement pymc-nightly==4.0.0b6.dev20220403 (from versions: 4.0.0b2.dev20220302, 4.0.0b2.dev20220303, 4.0.0b2.dev20220304, 4.0.0b3.dev20220308, 4.0.0b3.dev20220309, 4.0.0b3.dev20220310, 4.0.0b3.dev20220311, 4.0.0b3.dev20220312, 4.0.0b3.dev20220313, 4.0.0b3.dev20220314, 4.0.0b3.dev20220315, 4.0.0b3.dev20220316, 4.0.0b3.dev20220317, 4.0.0b4.dev20220318, 4.0.0b4.dev202203[19](https://github.com/pymc-devs/pymc/runs/5802654081?check_suite_focus=true#step:5:19), 4.0.0b4.dev[20](https://github.com/pymc-devs/pymc/runs/5802654081?check_suite_focus=true#step:5:20)220320, 4.0.0b4.dev20220321, 4.0.0b4.dev20220322, 4.0.0b5.dev20220323, 4.0.0b5.dev20220324, 4.0.0b5.dev20220325, 4.0.0b5.dev20220326, 4.0.0b5.dev20220327, 4.0.0b5.dev20220328, 4.0.0b5.dev20220329, 4.0.0b5.dev20220330, 4.0.0b6.dev20220331, 4.0.0b6.dev20220401, 4.0.0b6.dev20220402, 4.0.0b6.dev20220403)
19
ERROR: No matching distribution found for pymc-nightly==4.0.0b6.dev20220403
20
Error: Process completed with exit code 1.
```

Is what's shown here: https://github.com/pymc-devs/pymc/runs/5802654081?check_suite_focus=true

**Comment by theorashid** [comment]
> is this the same error I get in #5598 ? Because something is uploading to [pypi](https://pypi.org/project/pymc-nightly/4.0.0b6.dev20220401/)

I think it's the same issue as this. It's building something and uploading to pypi, but there's an issue with `WARNING: Generating metadata for package pymc-nightly produced metadata for project name pymc. Fix your #egg=pymc-nightly fragments.` Does anyone know how to fix this? I am unsure.

Also, reasking what I said on #5598:

I get the same errror (`WARNING: Generating metadata for package aesara-nightly produced metadata for project name aesara. Fix your #egg=aesara-nightly fragments.`) when I try `pip install aesara-nightly` (no test on aesara's nightly.yml so it wouldn't be picked up).

> Are nightly builds that useful if nobody spotted the aesara error yet? Can we not just use `pip install git+https://github.com/aesara-devs/aesara` and `pip install git+https://github.com/pymc-devs/pymc`? Why do we have to install directly into the repo at the moment?

**Comment by theorashid** [comment]
> Are nightly builds that useful if nobody spotted the aesara error yet? Can we not just use `pip install git+https://github.com/aesara-devs/aesara` and `pip install git+https://github.com/pymc-devs/pymc`? Why do we have to install directly into the repo at the moment?

Can I bump this? I use `pip install git+https://github.com/pymc-devs/pymc` myself and it works well. I think it complicates the codebase and pypi to have nightly. I'm happy to open PRs here and on aesara stripping both repos of nightly builds.

**Comment by ricardoV94** [comment]
Closed via #5713
